test-app/tests/integration/components/bs-modal/footer-test.gts

```diff
@@ -55,9 +60,9 @@ module('Integration | Component | bs-modal/footer', function (hooks) {
     assert
       .dom('.modal-footer button:first-child')
       .hasClass(defaultButtonClass(), 'Close button is a default button.');
-    assert.ok(
+    assert.strictEqual(
```

Comment:
We should refactor here to `assert.dom('.modal-footer button:first-child').hasAttribute('type',   'button', 'Submit button is of type submit.')` as well for readability at some point. But not necessarily in this PR.

Reply:
Changed it to make use of Qunit DOM